java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -1092,6 +1094,9 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         results.add(Pair.of(FunctionList, buildArgs(WITH_CODE_REDIS_API)));
         results.add(Pair.of(FunctionList, buildArgs(LIBRARY_NAME_REDIS_API, "*")));
 
+        transaction.functionDump();
+        results.add(Pair.of(FunctionDump, buildArgs()));
+
```

Comment:
Any creative method that I can test `functionRestore` here?

Reply:
I believe, this is just a unit test and can take in any mock arguments. Will need to do this in IT though and have two transactions in which one transaction uses the result of `functionDump` for the other transaction's `functionRestore`.

Reply:
It is fine, we just validate that command ID and argument list are corrent in this test.